app/models/katello/content_view.rb

```diff
@@ -111,6 +111,27 @@ class ContentView < Katello::Model
     scoped_search :on => :composite, :complete_value => true
     scoped_search :on => :generated_for, :complete_value => true
     scoped_search :on => :default # just for ordering
+    scoped_search :on => :name, :complete_value => true,
+                  :rename => :content_views,
+                  :operators => ['=', '!=', '~', '!~'],
```

Comment:
I filtered out the operators like `<` that don't make much sense here, as well as `^` which just don't seem to work well for this use case